go.mod

```diff
@@ -0,0 +1,83 @@
+module github.com/moby/buildkit
+
+go 1.11
+
+require (
+	github.com/BurntSushi/toml v0.3.1
+	github.com/Microsoft/go-winio v0.4.11
+	github.com/Microsoft/hcsshim v0.7.9 // indirect
+	github.com/apache/thrift v0.0.0-20161221203622-b2a4d4ae21c7 // indirect
+	github.com/codahale/hdrhistogram v0.0.0-20160425231609-f8ad88b59a58 // indirect
+	github.com/containerd/cgroups v0.0.0-20181219155423-39b18af02c41 // indirect
+	github.com/containerd/console v0.0.0-20180822173158-c12b1e7919c1
+	github.com/containerd/containerd v0.0.0-20181221132743-47b328aab791
+	github.com/containerd/continuity v0.0.0-20181001140422-bd77b46c8352
+	github.com/containerd/cri v0.0.0-20181012222137-f913714917d2 // indirect
+	github.com/containerd/fifo v0.0.0-20180307165137-3d5202aec260 // indirect
+	github.com/containerd/go-runc v0.0.0-20180907222934-5a6d9f37cfa3
+	github.com/containerd/typeurl v0.0.0-20180627222232-a93fcdb778cd // indirect
+	github.com/coreos/go-systemd v0.0.0-20181031085051-9002847aa142 // indirect
+	github.com/davecgh/go-spew v1.1.0 // indirect
+	github.com/docker/cli v0.0.0-20190128200251-cf6c238660e9
+	github.com/docker/distribution v0.0.0-20170825220652-30578ca32960
+	github.com/docker/docker v0.0.0-20180531152204-71cd53e4a197
+	github.com/docker/docker-credential-helpers v0.6.0 // indirect
+	github.com/docker/go-connections v0.3.0
+	github.com/docker/go-events v0.0.0-20170721190031-9461782956ad // indirect
+	github.com/docker/go-units v0.3.1 // indirect
+	github.com/docker/libnetwork v0.0.0-20180913200009-36d3bed0e9f4
+	github.com/godbus/dbus v4.1.0+incompatible // indirect
```

Comment:
The `+incompatible` just means that even though the repo is using semver, it doesn't have a go.mod, so go is precautious and treats it as v0 (aka things can break when it updates).